index.js

```diff
@@ -1,16 +1,17 @@
 var through = require('through');
-var riotCompiler = require('riot/compiler');
+var riotCompiler = require('riot/compiler/compile');
 
 module.exports = function (file, o) {
   var opts = o;
   var content = '';
+  var reqriot = 'var riot = require(\'riot\');';
```

Comment:
I figured since this module has a specific use case within a browserify environment it made sense, but yeah I think leaving it up to the caller is fine as well :D.